hipcheck-common/src/chunk.rs

```diff
@@ -223,9 +223,7 @@ mod test {
 		let res = match chunk_with_size(query, 10) {
 			Ok(r) => r,
 			Err(e) => {
-				println!("{e}");
-				assert!(false);
-				return;
+				panic!("{e}");
```

Comment:
Is `panic!` really correct to use in test cases?

Reply:
assert!(false) is raising a clippy warning. If there is an error and the test panics, the test harness will catch it and display the error to the user